crates/binjs_io/src/multipart/write.rs

```diff
@@ -1,23 +1,35 @@
+use TokenWriterError;
+
 use bytes;
-use bytes::compress::*;
-use bytes::varnum::*;
-use io::*;
-use ::TokenWriterError;
-use multipart::*;
+use bytes::compress::Compression;
+use bytes::compress::CompressionResult;
+use bytes::varnum::WriteVarNum;
+use io::TokenWriter;
+
+use multipart::FormatInTable;
+use multipart::HEADER_GRAMMAR_TABLE;
+use multipart::HEADER_STRINGS_TABLE;
+use multipart::HEADER_TREE;
 
 use std;
-use std::collections::{ HashMap, HashSet };
```

Comment:
Please don't change the coding style.

Reply:
Np.  Wasn't really thinking of it - just mostly committing the changes I was making to the import list as I tried to read through and understand the code.  I'll make sure to keep the existing multiple-name-import style.